test/snapshots/seam-blueprint.test.ts.md

```diff
@@ -103,40 +112,41 @@ Generated by [AVA](https://avajs.dev).
             },
             {
               deprecationMessage: '',
-              description: '',
+              description: 'Display name that corresponds to the brand-specific terminology for the `acs_system` type.',
               isDeprecated: false,
+              isUndocumented: false,
               name: 'external_type_display_name',
               type: 'string',
             },
             {
               deprecationMessage: '',
-              description: '',
+              description: 'Alternative text for the `acs_system` image.',
               isDeprecated: false,
+              isUndocumented: false,
               name: 'image_alt_text',
               type: 'string',
             },
             {
               deprecationMessage: '',
-              description: '',
+              description: 'URL for the image that represents the `acs_system`.',
               isDeprecated: false,
+              isUndocumented: false,
               name: 'image_url',
               type: 'string',
             },
             {
               deprecationMessage: '',
-              description: '',
+              description: 'Name of the `acs_system`.',
               isDeprecated: false,
+              isUndocumented: false,
               name: 'name',
               type: 'string',
             },
             {
               deprecationMessage: '',
-              description: `␊
-                    ---␊
-                    deprecated: use external_type␊
```

Comment:
Should this become the depractionMessage?

Reply:
Yes.